packages/web-components/src/dialog/dialog.stories.ts

```diff
@@ -0,0 +1,238 @@
+        efficitur auctor nibh, nec sollicitudin eros semper eget. Cras a elit ut tortor semper volutpat eu vel nunc.
+        Duis dapibus quam risus, ac tristique nisl aliquam eu. Curabitur vel ipsum non nunc euismod fringilla vel a
+        lorem. Curabitur viverra magna ac justo fringilla, eu vestibulum purus finibus. Donec elementum volutpat libero,
+        in tempus massa convallis vitae. Curabitur vitae mauris id urna dictum pharetra. Nullam vehicula arcu arcu,
+        vitae elementum enim tincidunt at. Duis eleifend, lorem a efficitur facilisis, nulla dolor finibus orci, et
+        ullamcorper orci ex ac purus. Aenean sem lectus, malesuada id magna id, facilisis condimentum nibh. Cras tempor
+        neque mi, sit amet suscipit libero consectetur non. Nullam id eleifend mauris. Mauris iaculis lectus eu
+        scelerisque efficitur. In id suscipit libero. Donec condimentum, purus ac laoreet facilisis, risus lorem
+        facilisis neque, id volutpat felis mi eget metus. Nulla facilisi. Donec consequat tincidunt nunc sed elementum.
+        Integer consectetur tristique orci, ut congue justo pellentesque eu. Fusce faucibus iaculis mauris, eu lobortis
+        orci egestas eget. Nullam nec arcu bibendum, cursus diam ac, facilisis enim. Nulla facilisi. Curabitur lacinia
+        odio mauris, a gravida nisi volutpat in. Aliquam at maximus felis. Vestibulum convallis dignissim urna id
+        gravida.
+      </div>
+      <fluent-button
+        slot="actions"
+        appearance="primary"
+        @click="${(e: Event, c) => closeDialog(e, 'dialog-longcontent')}"
+        >Close Dialog</fluent-button
+      >
+      <fluent-button slot="actions">Do Something</fluent-button>
+    </fluent-dialog>
+  </div>
+`);
+
+export const NoFocusTrap = renderComponent(html<DialogStoryArgs>`
+  <div>
+    <fluent-button @click=${(e: Event) => openDialog(e, 'dialog-nofocustrap')}>Open Dialog</fluent-button>
+    <fluent-dialog id="dialog-nofocustrap" no-focus-trap hidden>
+      <div slot="title">No Focus Trap</div>
+      <fluent-button
+        appearance="transparent"
+        icon-only
+        @click="${(e: Event, c) => closeDialog(e, 'dialog-nofocustrap')}"
+        slot="close"
+      >
+        ${dismissed16Regular}
+      </fluent-button>
+      <div>
+        The no-focus-trap attribute removes the default behavior of trapping focus within the dialog, granting users
+        greater flexibility in managing focus and interacting with other elements on the page.
+      </div>
+      <br />
+      <fluent-text><code>fluent-dialog no-focus-trap</code></fluent-text>
+
+      <fluent-button
+        slot="actions"
+        appearance="primary"
+        @click="${(e: Event, c) => closeDialog(e, 'dialog-nofocustrap')}"
+        >Close Dialog</fluent-button
+      >
+      <fluent-button slot="actions">Do Something</fluent-button>
+    </fluent-dialog>
+  </div>
+`);
```

Comment:
Also show some "no trap focus" scenarios.

Reply:
I added the additional stories. `no-trap-focus` is no longer pertinent since I've switched to using the native dialog element. The original intention of `no-focus-trap` was to mimic the default behavior of the native dialog.